Answers.md

```diff
@@ -1 +1,17 @@
+1. Describe the biggest difference between .forEach & .map.
+The biggest difference is .map has a return value where .forEach does not. Map is creating a new array of the same length as the input array with new values. ForEach is only looping over each value.
 
+2. What is the difference between a function and a method?
+The difference between a function and a method is that functions are considered objects while methods are considered values of an object property but they are also functions. A method is usually called using dot notation: obj.myMethod().
+
+3. What is closure?
+Closures are functions that remember the free variables that were defined in their parent function.
+
```

Comment:
Yes and they are protected my the scope of the function they are defined in, meaning outer functions can't reference them

Still correct though ✅